Packet++/src/Asn1Codec.cpp

```diff
@@ -12,7 +15,106 @@
 #undef max
 #endif
 
-namespace pcpp {
+namespace pcpp
+{
+	std::string toString(Asn1TagClass tagClass)
+	{
+		switch (tagClass)
+		{
+			case Asn1TagClass::Universal:
+				return "Universal";
+			case Asn1TagClass::ContextSpecific:
+				return "ContextSpecific";
+			case Asn1TagClass::Application:
+				return "Application";
+			default:
+				return "Private";
+		}
+	}
+
+	std::string toString(Asn1UniversalTagType tagType)
+	{
+		switch (tagType)
+		{
+			case Asn1UniversalTagType::EndOfContent:
```

Comment:
recommend to use a hash table here.

Reply:
Fixed in this commit: https://github.com/seladb/PcapPlusPlus/pull/1403/commits/c2fffbeed339314526d9eb4b7d3b4bdd084657a6